encoding/src/commonMain/kotlin/diglol/encoding/internal/commonBase45.kt

```diff
@@ -0,0 +1,121 @@
+package diglol.encoding.internal
+
+import kotlin.native.concurrent.SharedImmutable
+
+/**
+ * Implementation of Base45 encoding/decoding.
+ *
+ * Basically, Base45 is just encode 2 bytes based on 256 into 3 bytes based on 45.
+ *
+ * It will result in a encoded string which will generate a compat QR code when using alphanumeric mode.
+ *
+ * @see [RFC 9285](https://datatracker.ietf.org/doc/rfc9285/)
+ */
+
+@SharedImmutable
+internal val BASE45 = byteArrayOf(
+  48, 49, 50, 51, 52, 53, 54, 55, 56, 57, // 0-9
+  65, 66, 67, 68, 69, 70, 71, 72, 73, 74, // A-J
+  75, 76, 77, 78, 79, 80, 81, 82, 83, 84, // K-T
+  85, 86, 87, 88, 89, 90, // U-Z
+  // SP, $, %, *, +, -, ., /, :
+  32, 36, 37, 42, 43, 45, 46, 47, 58
+)
+
+internal fun ByteArray.commonEncodeBase45(map: ByteArray = BASE45): ByteArray {
```

Comment:
It is recommended to remove the `map` parameter, because base45 has only one type.  ref: [Hex](https://github.com/diglol/encoding/blob/trunk/encoding/src/commonMain/kotlin/diglol/encoding/internal/commonHex.kt),  What do you think? Also please replace `Base45(Std)` with `Base45` in `README.md`.

Reply:
Yes.

I do think the map parameter is unnecessary, using this param is just wanting to align with the others.

And maybe someday there will be a URL_SAFE version of Base45.

But, as the saying goes, ***Premature optimization is the root of all evil***.

Therefore, I will remove the `map` param

Reply:
Fixed in e7a10ed5